app/api/helpers/system_mails.py

```diff
@@ -228,4 +229,9 @@ def entries():
         'subject': 'Test Mail Subject',
         'message': ("This is a  <strong> Test </strong> E-mail."),
     },
+    MailType.VIDEO_MODERATOR_INVITE: {
+        'recipient': 'User',
+        'subject': 'Video Moderator Invite',
```

Comment:
Match subject style to event role mail

Reply:
How do i get the role or it needs to be harcoded @iamareebjamal

Reply:
Role is only Video Moderator